### PR TITLE
Show tier in Compare box for Hedvig option

### PR DESCRIPTION
**Pull request by robinandeer**
## What?

Show which tier is selected in the compare box next to the "old insurance company".

Show date label for renewal date for the "old company" in compare box.

## Why?

So it's easier to see what you are comparing :)

**Ticket(s): []**

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* master
  * **PR #1185** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/web-onboarding/1185" target="_blank">Graphite</a> 
    * **PR #1188** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/web-onboarding/1188" target="_blank">Graphite</a>  👈
    * **PR #1186** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/web-onboarding/1186" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/web-onboarding/1188?utm_source=stack-comment).

File: src/client/pages/Offer/Introduction/ExternalInsuranceProvider/Compare.tsx [hunk 1]
```diff
@@ -70,15 +72,36 @@ const CompareBoxDescription = styled.div({
...
 }
 
-export const Compare: React.FC<Props> = ({ insuranceDataCollection, cost }) => {
+export const Compare: React.FC<Props> = (props) => {
+  const textKeys = useTextKeys()
+  const { insuranceDataCollection, cost, description } = props
   const translateCoverage = useTranslateCoverage()
+  const getDateLabel = useGetDateLabel()
+
+  const compareDescriptions = [
```
Comment (alebedev) [inline src/client/pages/Offer/Introduction/ExternalInsuranceProvider/Compare.tsx:93]: Nit: log of magic to build an array conditionally.  Could be simplified as 
```
const compareDesciptions = []
if (insuranceDataCollection.renewalDate) {
  compareDescriptions.push(...)
}
if (insuranceDataCollection.coverage) {
  compareDescriptions.push(...)
}
```

**Review by alebedev — APPROVED** [review]
Looks good, minor questions inside

**Review by guilhermespopolin — APPROVED** [review]

File: src/client/pages/Offer/Introduction/Sidebar/StartDate.tsx [hunk 2]
```diff
@@ -513,7 +513,7 @@ export const useStartDateProps = (): Omit<StartDateProps, 'size' | 'modal'> => {
   }
 }
 
-const useGetDateLabel = () => {
+export const useGetDateLabel = () => {
```
[the remark this answers is not in the document]
Comment (robinandeer) [reply src/client/pages/Offer/Introduction/Sidebar/StartDate.tsx:516]: hmm but it's using other hooks inside...

File: src/client/pages/Offer/Introduction/ExternalInsuranceProvider/Compare.tsx (hunk 1 above)
Comment (robinandeer) [reply src/client/pages/Offer/Introduction/ExternalInsuranceProvider/Compare.tsx:93]: fair enough :)

**Pull request merged**